lib/assets/stylesheets/govuk-frontend/README.md

```diff
@@ -0,0 +1,75 @@
+# The `src` directory
```

Comment:
Ideally GOV.UK Frontend would be installed via `npm install` to avoid having to vendor it in.

If that's not possible, I strongly recommend using the `package` folder from the repo rather than the contents of `source`, as the Sass in source is not vendor-prefixed, and the source folder includes e.g. all the tests.